ui/src/journals/components/JournalItem.tsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { Row, Col, PageHeader } from 'antd';
+import { Link } from 'react-router-dom';
+
+import ResultItem from '../../common/components/ResultItem';
+import EditRecordAction from '../../common/components/EditRecordAction';
+import AuthorizedContainer from '../../common/containers/AuthorizedContainer';
+import { SUPERUSER_OR_CATALOGER } from '../../common/authorization';
+import { JOURNALS_PID_TYPE } from '../../common/constants';
+import UrlsAction from '../../literature/components/UrlsAction';
+import { JOURNALS } from '../../common/routes';
+import { JournalMetadata } from '../containers/SearchPageContainer';
+
+export const JournalItem = ({
+  metadata,
+  isCatalogerLoggedIn,
+}: {
+  metadata: {
+    get: (arg: string) => JournalMetadata;
+    getIn: (arg: string[]) => JournalMetadata;
+  };
+  isCatalogerLoggedIn: boolean
+}) => {
+  const shortTitle = metadata.get('short_title');
+  // @ts-ignore
+  const publisher = metadata.get('publisher').toArray();
+  const journalTitle = metadata.getIn(['journal_title', 'title']);
+  const urls = metadata.get('urls');
+  const recordId = metadata.get('control_number').toString();
+
+  return (
+    <ResultItem
+      leftActions={
+        <>
+          {urls && <UrlsAction urls={urls} text="links" />}
+          <AuthorizedContainer authorizedRoles={SUPERUSER_OR_CATALOGER}>
+            <EditRecordAction pidType={JOURNALS_PID_TYPE} pidValue={recordId} isCatalogerLoggedIn={isCatalogerLoggedIn} />
+          </AuthorizedContainer>
+        </>
+      }
+    >
+      <Row>
+        <Col>
+          <Link className="result-item-title" to={`${JOURNALS}/${recordId}`}>
+            <PageHeader
+              className="site-page-header"
+              title={shortTitle}
+              subTitle={`(${publisher.map((p: string) => p)})`}
+            />
+          </Link>
+        </Col>
+      </Row>
+      <Row>
+        <Col>{journalTitle}</Col>
+      </Row>
+    </ResultItem>
```

Comment:
`papers_count` is missing

Reply:
This will be added in second step of completing the task ([explained here](https://github.com/cern-sis/issues-inspire/issues/5#issuecomment-1160234416))